experimental/PropertyDDS/packages/property-dds/src/propertyTreeExt.ts

```diff
@@ -20,3 +20,13 @@ export class DeflatedPropertyTree extends SharedPropertyTree {
         return new DeflatedPropertyTreeFactory();
     }
 }
+
+export class LZ4PropertyTree extends SharedPropertyTree {
+    public static create(runtime: IFluidDataStoreRuntime, id?: string, queryString?: string) {
+        return runtime.createChannel(id, DeflatedPropertyTreeFactory.Type) as LZ4PropertyTree;
```

Comment:
This should probably be LZ4PropertyTreeFactory.Type?

Reply:
@DLehenbauer thank you for finding this